DOC-1941: Copy edit changes to changelog.md 6.4.1 (#2676)
tinymce/tinymce-docs: modules/ROOT/pages/changelog.adoc

=== Added
* New `tree` component that can be used in dialog body panel.
* `renderUI` property in the `Theme` type can now return a `Promise<RenderResult>` instead of `RenderResult`.
* New `editor.selection.isEditable` API method that returns whether the current selection is editable.
* New `editor.dom.isEditable` API method that returns whether the specified node is editable.
* New `setText` and `setIcon` methods added to menu button and toolbar button APIs.
* New `highlight_on_focus` option which enables highlighting the content area on focus.
* New `fontsizeinput` toolbar item which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons.

> * Added `skip_focus` check to the ToggleToolbarDrawer command.

Reviewer: No idea what `skip_focus` is

Reviewer 2: its an optional argument for certain commands which involve UI elements, when false or null the editor will be focused after the command is run, but when set to true this focus step will be skipped.

> * New `font_size_input_default_unit` option allows to use of numbers without a unit in `fontsizeinput` and have them parsed with the default unit, if it is not defined the default is `pt`.

Reviewer: ```suggestion
* New `font_size_input_default_unit` option allows to use of numbers without a unit in `fontsizeinput` and have them parsed with the default unit, if it is not defined.  The default unit is `pt`.
```

Author: Closing without commiting suggestion as discussed.

> * New `group` and `togglebutton` in view.

Reviewer: Are these new buttons or properties?

Author: Closing without committing suggestion as discussed.

* Updated focus, active, and enabled colors of UI buttons for improved contrast against the UI color.

=== Changed
* The Link plugin's context menu items will no longer appear for `contenteditable="false"` links.
* The formatting of elements with a `contenteditable="false"` attribute are no longer cloned to new cells when new table rows are created.
* Changed the color of `@dialog-table-border-color`, and added right padding to the first cell of dialog table.

=== Fixed
* Sometimes the editor would finish initializing before the theme would have finished loading.
* The `searchreplace` modal closed incorrectly when clicking outside of the alert that pops up when no match is found.
* The text color or background color picker toolbar buttons did not update when the text color or background color was changed using the equivalent commands in the Format menu.
* The `onSetup` api function would not run when defining custom group toolbar button.
* The foreground and background menu icons would not properly update to display the last used color.
* Added new `setIconFill` function to `editor.ui.addNestedMenuItem`.

> * Pasting links to text would sometimes not generate the correct undo stack on Safari.

Reviewer: ```suggestion
* Pasting links to text would sometimes cause the incorrect undo state on Safari.
```

????

Author: Closing without committing suggestion as discussed.

* Textareas with scrollbars in dialogs would not render rounded corners correctly on some browsers.
* It was possible to open links inside the editor if the editor root was an element with a `contenteditable="false"` attribute.
* Inline boundary was rendered for boundary elements that had a `contenteditable="false"` attribute.

> * Clicking on a disabled split button would call the onAction callback.

Reviewer: ```suggestion
* Clicking on a disabled split button would call the `onAction` callback.
```
